jenkins/table_jenkins_user.go

```diff
@@ -0,0 +1,45 @@
+package jenkins
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+)
+
+//// TABLE DEFINITION
+
+func tableJenkinsUser() *plugin.Table {
+	return &plugin.Table{
+		Name:        "jenkins_user",
+		Description: "An extension to Jenkins functionality provided separately from Jenkins Core.",
+		List: &plugin.ListConfig{
+			Hydrate: getlistJenkinsUsers,
+		},
+
+		Columns: []*plugin.Column{
```

Comment:
> @IvenGe could we add the other columns mentioned in https://github.com/IvenGe/gojenkins/blob/master/users.go#L32-L38?
> 
> Is there a specific reason why we omitted those columns?

I kept it to query one single user https://github.com/IvenGe/gojenkins/blob/f58de906ee88b1487bc92eeb07f9cbb9f344e923/users.go#L119
In case I need it later.